src/Data/DotLang/Attr/Node.purs

```diff
@@ -115,3 +125,19 @@ instance dotLangShape :: DotLang ShapeType where
   toText Rarrow = "Rarrow"
   toText Larrow = "Larrow"
   toText Lpromoter = "Lpromoter"
+
+-- |
+-- | ```purescript
+-- | htmlLabel "<table><tr><td>Label</td></tr></table>" -- :: Attr
+-- | ```
+-- | htmlLabel as a part of an attribute of a node.
+htmlLabel :: String -> Attr
+htmlLabel = HtmlLabel >>> Label
+
+-- |
+-- | ```purescript
+-- | textLabel "..." -- :: Attr
+-- | ```
+-- | textLabel as a part of an attribute of a node.
+textLabel :: String -> Attr
+textLabel = HtmlLabel >>> Label
```

Comment:
the same here

Reply:
Made the changes, also noticed I had implemented it incorrectly; both of them were making html labels. Will probably add a test in a new PR.